translations/harbour-barcode-es.ts

```diff
@@ -6,12 +6,12 @@
     <message id="about-title">
         <source>About CodeReader</source>
         <extracomment>About page title, label and menu item</extracomment>
-        <translation>Acerca de CodeReader</translation>
+        <translation>Acerca de</translation>
     </message>
     <message id="about-description">
         <source>This app demonstrates a bar code reader for Sailfish OS. I hope it is useful for other projects. CodeReader is open source and licensed under the MIT License.</source>
         <extracomment>About page text</extracomment>
-        <translation>Esta aplicación muestra un lector de código de barras para el sistema operativo SailfishOS. Espero que sea útil para otros proyectos. CodeReader es de código abierto y está registrado bajo la licencia MIT.</translation>
+        <translation>Esta aplicación muestra un lector de código de barras para el sistema operativo Sailfish SO. Espero que sea útil para otros proyectos. CodeReader es de código abierto y está registrado bajo la licencia MIT.</translation>
```

Comment:
Sailfish SO?

Reply:
SO is not stick with sailfish directly, because SO is stand of Operating Sistem, in this case in spanish is SO Sistema Operativo.